static/js/searchPage.js

```diff
@@ -67,3 +72,26 @@ function showRestaurants() {
     restaurantContainer.appendChild(restaurantDiv);
   });
 }
+
+// Initializes a map
+function initMap() {
+	const map = new google.maps.Map(
+		document.getElementById('map'), {
+			center: {
+				lat: 46.2276,
+				lng: 2.2137
+			},
+			zoom: 6
+		});
+
+	var iconBase = 'https://maps.google.com/mapfiles/kml/shapes/';
```

Comment:
Also, Vedant commented on one of my PRs about avoiding 'var' and using 'let' and 'const' instead. This is a suggestion in Google's style guide, and I think it helps avoid possible unnecessary scoping issues. I don't think that is an issue here, but it is worth noting for the future I think.
https://google.github.io/styleguide/jsguide.html#features-local-variable-declarations